sms_statistics_service/sms_statistics_service.py

```diff
@@ -99,6 +99,8 @@
             if msg.urn.startswith("tel:"):
                 # Set the operator name from the phone number
                 operator_name = PhoneCleaner.clean_operator(msg.urn)
+            elif msg.urn.startswith("deleted:"):
+                operator_name = Codes.NOT_CODED
```

Comment:
NC or really actually missing?

Reply:
As discussed offline, we'll change this to Codes.DELETED. 

This requires adding Codes.DELETED to Core, which is currently hard because there are outstanding PRs in Core that change how the control codes are used. To avoid implementing this twice because a PR is open, I propose delaying this PR until those are closed.